torch/_dynamo/symbolic_convert.py

```diff
@@ -876,8 +876,7 @@ def pop(self) -> VariableTracker:
         return self.stack.pop()
 
     def popn(self, n: int) -> List[VariableTracker]:
-        assert n >= 0
-        return list(reversed([self.pop() for _ in range(n)]))
+        return [*reversed([self.pop() for _ in range(n)])]
```

Comment:
Huh, is this really faster than `list` of reverse iterator?? I would expect the `return list(reversed([self.pop() for _ in range(n)]))` to be faster do the loop in C code.

Reply:
It gets mapped to the `LIST_EXTEND` bytecode:
```
>>> from dis import dis
>>> dis(lambda x: list(x))
  1           0 RESUME                   0
              2 LOAD_GLOBAL              1 (NULL + list)
             14 LOAD_FAST                0 (x)
             16 PRECALL                  1
             20 CALL                     1
             30 RETURN_VALUE
>>> dis(lambda x: [*x])
  1           0 RESUME                   0
              2 BUILD_LIST               0
              4 LOAD_FAST                0 (x)
              6 LIST_EXTEND              1
              8 RETURN_VALUE
>>> timeit.timeit("list(range(3))")
0.12507863098289818
>>> timeit.timeit("[*range(3)]")
0.1083272109972313
```